user-guide/Cloud_Platform/Catalog/Deploying_A_DataMiner_Connector_to_your_system.md

```diff
@@ -39,4 +39,4 @@ To deploy an item from the DataMiner Catalog (e.g. a connector or package) to yo
    The item will be pushed to the DataMiner System. In the Admin app, you can check the status of the deployment. See [Viewing information on deployments](xref:Viewing_info_on_deployments).
 
 > [!NOTE]
-> Deploying a connector from the Catalog will never change the production version for that connector in the DataMiner System.
+> From DataMiner 10.4.10/10.5.0 onwards<!--RN 40291-->, when you install a connector for the first time by deploying it from the DataMiner Catalog, it will automatically be promoted to the production version. This action adds the following entry to the *SLNet.txt* log file: `Initial upload of protocol '{protocolName}'. Version '{protocolVersion}' will be automatically set as production.` However, when you later deploy a new version of the same connector and want it to be set as the production version, you will need to manually [promote this protocol version](xref:Promoting_a_protocol_version_to_production_version). Prior to DataMiner 10.4.10/10.5.0, deploying a connector from the Catalog will never change the production version for that connector in the DataMiner System.
```

Comment:
Same here
```suggestion
> From DataMiner 10.4.10/10.5.0 onwards<!--RN 40291-->, when you install a connector for the first time by deploying it from the DataMiner Catalog, it will automatically be promoted to the production version. However, when you later deploy a new version of the same connector and want it to be set as the production version, you will need to manually [promote this protocol version](xref:Promoting_a_protocol_version_to_production_version). Prior to DataMiner 10.4.10/10.5.0, deploying a connector from the Catalog will never change the production version for that connector in the DataMiner System.
```